react/tic-tac-toe/src/app/screens/Game/components/Matches/index.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import { Redirect} from 'react-router-dom';
+import styles from '../../styles.module.scss';
+import Navbar from "../Navbar"
+import { connect } from "react-redux";
+import actionCreators from "../../../../../redux/actions";
+import Spinner from 'react-spinkit';
+
+class Matches extends Component {
+  
+  async componentDidMount(){
+    this.props.dispatch(actionCreators.getMatches());
+  }
+
+  render() {
+    if(!window.localStorage.getItem("token")){
+      return <Redirect to="/"/>;
+    }
+
+    const {matches} = this.props;
+
+    if(this.props.isLoading){
+      return <Spinner name='folding-cube' />;
+    }
+
+    if(this.props.error){
```

Comment:
Add `isLoading` and `error ` here:   `const {matches} = this.props;`

Reply:
:white_check_mark: